modules/custom/az_migration/config/install/migrate_plus.migration.az_media.yml

```diff
@@ -0,0 +1,31 @@
+id: az_media
+label: All Media (media_migration)
+migration_group: az_migration
+audit: true
+migration_tags:
+  - Drupal 7
+  - Content
+deriver: Drupal\media_migration\Plugin\migrate\D7FileEntityDeriver
+source:
+  plugin: d7_file_entity_item
+process:
+  uuid:
+    -
+      plugin: media_migrate_uuid
+      source: fid
+    -
+      plugin: skip_on_empty
+      method: process
+  mid: fid
```

Comment:
```suggestion
```

Reply:
I don't think this should be present. `mid` and `fid` are not the same identifiers.